index.html

```diff
@@ -10,13 +10,16 @@
 <html>
 <head>
 
-  <script src="../webcomponentsjs/webcomponents.js"></script>
-  <link rel="import" href="../core-component-page/core-component-page.html">
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <script src="../webcomponentsjs/webcomponents-lite.js"></script>
+  <link rel="import" href="../iron-doc-viewer/iron-doc-viewer.html">
```

Comment:
Let's add `polymerelements/iron-doc-viewer#^0.8.0` to the bower `devDependencies`.